zng/docs/zson.md

```diff
@@ -257,41 +276,45 @@ The syntax for a decorator is a parenthesized type, as in
 ```
 <value> ( <decorator> )
 ```
-where a `<decorator>` is either an integer representing an internal type
-or a ZSON type value.  Note that an internal type is not a ZSON type value.
+where a `<decorator>` is either a type or a type definition.
 
 It is an error for the decorator to be type incompatible with its referenced value.  
 
 > TBD: define precisely the notion of type compatibility
 
 #### 3.2.1 Type Definitions
 
-Type names are defined by binding a name to a type with an assignment decorator
+New type names are created by binding a name to a type with an assignment decorator
 of the form
 ```
 <value> (= <identifier> )
 ```
-This creates a new type whose name is given by the identifier.
-The type bound to that name is the type value of `<value>`.  This new
-type may then be used as or within a type value or decorator.  The name
+This creates a new type whose name is given by the identifier and whose
+type is equivalent to the type of `<value>`.  This new
+type may then be used anywhere a type may appear.  The name
 of the type definition
 must not be equal to any of the primitive type names.
 
-The result of a type definition is a value with the new type.
-Thus, type definitions can be chained as in
+A type definition may also appear inside of decorator as in
 ```
- <value> (= <identifier> ) (= <identifer> )
+<identifier> (= <type> )
```

Comment:
Is it deliberate that the word "alias" doesn't appear here?

Reply:
Oops, that's supposed to be
```
<identifier> = ( <type> )
```
And yes I'm intentionally avoiding "alias" in favor of "type name" in the ZSON spec.  I will leave "alias" as the low-level concept in zng that implements user-defined ZSON type names.  Does this seem reasonable?

Reply:
Yes, it sounds reasonable to me